suite-native/module-send-receive/src/screens/ReceiveAccountsScreen.tsx

```diff
@@ -1,33 +1,22 @@
 import React from 'react';
 
-import { useNavigation } from '@react-navigation/native';
-
 import { AccountsList } from '@suite-native/accounts';
 import {
-    AppTabsParamList,
-    AppTabsRoutes,
-    RootStackRoutes,
     Screen,
     ScreenHeader,
-    RootStackParamList,
-    TabToStackCompositeNavigationProp,
+    SendReceiveStackParamList,
+    SendReceiveStackRoutes,
+    StackProps,
 } from '@suite-native/navigation';
 
-type ReceiveAccountsNavigationProps = TabToStackCompositeNavigationProp<
-    AppTabsParamList,
-    AppTabsRoutes.ReceiveScreen,
-    RootStackParamList
->;
-
-export const ReceiveAccountsScreen = () => {
-    const navigation = useNavigation<ReceiveAccountsNavigationProps>();
-    return (
-        <Screen header={<ScreenHeader title="Receive to" hasGoBackIcon={false} />}>
-            <AccountsList
-                onSelectAccount={accountKey =>
-                    navigation.navigate(RootStackRoutes.ReceiveModal, { accountKey })
-                }
-            />
-        </Screen>
-    );
-};
+export const ReceiveAccountsScreen = ({
+    navigation,
+}: StackProps<SendReceiveStackParamList, SendReceiveStackRoutes.ReceiveAccounts>) => (
+    <Screen header={<ScreenHeader title="Receive to" hasGoBackIcon={false} />}>
+        <AccountsList
+            onSelectAccount={accountKey =>
+                navigation.navigate(SendReceiveStackRoutes.Receive, { accountKey })
+            }
```

Comment:
nit: Can you please move this to handler function? I know it's been like that but small improvement